patches/minecraft/net/minecraft/enchantment/Enchantment.java.patch

```diff
@@ -48,3 +59,20 @@
      public static void func_185257_f()
      {
          EntityEquipmentSlot[] aentityequipmentslot = new EntityEquipmentSlot[] {EntityEquipmentSlot.HEAD, EntityEquipmentSlot.CHEST, EntityEquipmentSlot.LEGS, EntityEquipmentSlot.FEET};
+@@ -209,4 +233,16 @@
+             return this.field_185275_e;
+         }
+     }
++
++    /* ======================================== FORGE START =====================================*/
++
++    /**
++     * Calculate the damage this enchantment adds to an attack on the given entity
++     */
```

Comment:
Add a period and

```
@param level the level of this enchantment on the item doing the attack
@param entity the entity being attacked
@return the additional damage to do
```

to the Javadoc (preserving case exactly, so as to conform to the convention).